RealmSwift/AsymmetricObject.swift

```diff
@@ -61,7 +61,7 @@ extension AsymmetricObject: _RealmCollectionValueInsideOptional {
 
      - parameter value:  The value used to populate the object.
      */
-    public convenience init(value: Any) {
+    internal convenience init(value: Any) {
```

Comment:
This should be public. Unmanaged asymmetric objects are a little weird, but they are something we allow.